packages/react-signature-frame/package.json

```diff
@@ -5,12 +5,14 @@
   "author": "Anvil Foundry Inc.",
   "license": "MIT",
   "main": "dist/index.js",
+  "types": "./types/index.d.ts",
   "scripts": {
     "build": "webpack --mode production",
     "build:changelog": "yarn auto-changelog",
     "pack": "yarn pack",
     "version": "yarn build:changelog && git add CHANGELOG.md",
-    "test": "yarn mocha --config ./test/mocha.js"
+    "test": "yarn mocha --config ./test/mocha.js",
+    "tsc": "tsc"
```

Comment:
This necessary?

Reply:
Yeah suppose not. I actually need to add that into the build script instead.